datadriven/src/sgpp/datadriven/datamining/modules/fitting/ModelFittingDensityDifferenceEstimationOnOffParallel.cpp

```diff
@@ -217,6 +217,43 @@ void ModelFittingDensityDifferenceEstimationOnOffParallel::update(DataMatrix& ne
   }
 }
 
+double ModelFittingDensityDifferenceEstimationOnOffParallel::computeResidual(
+    DataMatrix& validationData) const {
+  DataVectorDistributed bValidation = online->computeWeightedDerivativeBFromBatchParallel(
+      validationData, *grid, this->config->getDensityEstimationConfig(),
```

Comment:
@janschopohl isn't this DensityDifference still? You seem to be calling the rhs for the DensityDerivative...